modules/test/playwright/tests/journal-web/journalArticle.spec.ts

```diff
@@ -28,6 +29,75 @@ const PERMISSIONS_LOCATORS = [
 	'#guest_ACTION_PERMISSIONS',
 ];
 
+const expect = baseExpect.extend({
+	toBeSuccessful: (response: APIResponse) => ({
+		message: () =>
+			response.ok()
+				? 'Response is successful'
+				: 'Response is not successful',
+		pass: response.ok(),
+	}),
+});
+
+test('LPD-6813: Make prefix URLs configurable', async ({
+	friendlyUrlInstanceSettingsPage,
+	journalEditArticlePage,
+	journalPage,
+	page,
+	pageTemplatePage,
+}) => {
+	await journalPage.goto();
+
+	const articleTitle = getRandomString();
+
+	await journalEditArticlePage.publishNewBasicArticle(articleTitle);
+
+	const article = page
+		.locator(
+			'#_com_liferay_journal_web_portlet_JournalPortlet_articlesSearchContainer .list-group-item'
+		)
+		.filter({hasText: articleTitle});
+
+	await article.waitFor();
+
+	await pageTemplatePage.goToDisplayPageTemplates();
+
+	const displayPageTemplateName = getRandomString();
+
+	await pageTemplatePage.publishNewDisplayPageTemplate(
+		displayPageTemplateName
+	);
+
+	await pageTemplatePage.markPageTemplateAsDefault(displayPageTemplateName);
+
+	await friendlyUrlInstanceSettingsPage.goto();
+
+	const urlSeparator = 'content';
+
+	await friendlyUrlInstanceSettingsPage.modifySeparator(
+		'_com_liferay_configuration_admin_web_portlet_InstanceSettingsPortlet_com.liferay.journal.model.JournalArticle',
+		urlSeparator
+	);
+
+	expect(
+		await page.request.get('/' + urlSeparator + '/' + articleTitle)
```

Comment:
this page.request expect statement is failing on my side. Could you take a look a please?